src/module.c

```diff
@@ -421,8 +415,10 @@ int DropIndexCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return RedisModule_WrongArity(ctx);
   }
 
-  IndexSpec *sp = IndexSpec_Load(ctx, RedisModule_StringPtrLen(argv[1], NULL), 0);
-  if (sp == NULL) {
+  const char* spec_name = RedisModule_StringPtrLen(argv[1], NULL);
+  StrongRef global_ref = IndexSpec_LoadUnsafe(ctx, spec_name, 0);
```

Comment:
What the name global stands for? Why not spec_ref or spec_strong_ref?

Reply:
because it is a borrow of the global strong ref. can change it